src/main/blackbox/blackbox.c

```diff
@@ -873,26 +901,24 @@ static bool blackboxWriteSysinfo()
             //Shouldn't ever get here
         break;
         case 1:
-            blackboxPrintf("H Firmware type:Cleanflight\n");
-
-            xmitState.u.serialBudget -= strlen("H Firmware type:Cleanflight\n");
+            xmitState.u.serialBudget -= blackboxPrint("H Firmware type:Cleanflight\n");
         break;
         case 2:
             blackboxPrintf("H Firmware revision:%s\n", shortGitRevision);
 
-            /* Don't need to be super exact about the budget so don't mind the fact that we're including the length of
-             * the placeholder "%s"
-             */
-            xmitState.u.serialBudget -= strlen("H Firmware revision:%s\n") + strlen(shortGitRevision);
+            xmitState.u.serialBudget -= strlen("H Firmware revision:\n") + strlen(shortGitRevision);
         break;
         case 3:
             blackboxPrintf("H Firmware date:%s %s\n", buildDate, buildTime);
 
-            xmitState.u.serialBudget -= strlen("H Firmware date:%s %s\n") + strlen(buildDate) + strlen(buildTime);
+            xmitState.u.serialBudget -= strlen("H Firmware date: \n") + strlen(buildDate) + strlen(buildTime);
```

Comment:
The space here isn't consistent with the other `blackboxPrintf()` calls above and below.